guides/v2.3/extension-dev-guide/prepare/ext-repo-structure.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: default
+group: extension-dev-guide
+title: Extension Repository Structure
+version: 2.3
+github_link: extension-dev-guide/prepare/ext-repo-structure.md
+---
+
+## Extension Repository Structure
+
+To create Magento 2.3 extensions for modules, themes, and language packs, your extension repository should adhere to these five changes:
```

Comment:
I'd rephrase "should" to something less strict. Like "we recommend". Because in reality they can use whatever structure they want.
Unless this is documentation specifically for CBE developers.